src/clients/c++/perf_client.cc

```diff
@@ -1665,10 +1719,17 @@ Usage(char** argv, const std::string& msg = std::string())
       << std::endl;
   std::cerr
       << "For --percentile, it indicates that the specified percentile in terms"
-      << " of latency will be reported and used to detemine if the measurement"
-      << " is stable instead of average latency."
-      << " Default is -1 to indicate no percentile will be used or reported."
-      << std::endl;
+      << " of latency will also be reported and used to detemine if the"
+      << " measurement is stable instead of average latency."
+      << " Default is -1 to indicate no percentile will be used." << std::endl;
+  std::cerr
+      << "For --data-directory, it indicates that the perf client will use user"
+      << " provided data instead of synthetic data for model inputs. For each"
+      << " input, there must be a binary file in the way that its relative path"
```

Comment:
"... must be a binary file for each input required by the model. The file must be named the same as the input and must contain data required for sending the input...